include/service/cache_entity_service.js

```diff
@@ -52,16 +55,14 @@ module.exports = function CacheEntityServiceModule(pb) {
     CacheEntityService.prototype.get = function(key, cb){
 
         var self = this;
-        pb.cache.get(key, function(err, result){
+        pb.cache.get(key, function(err, result){console.log(util.inspect(result));
```

Comment:
Fail... No, I thought I got them all. Thank you for the heads up.
On May 21, 2015 2:47 PM, "Mark Baird" <notifications@github.com> wrote:

> In include/service/cache_entity_service.js
> <https://github.com/pencilblue/pencilblue/pull/620#discussion_r30834108>:
>
> > @@ -52,16 +55,14 @@ module.exports = function CacheEntityServiceModule(pb) {
> >      CacheEntityService.prototype.get = function(key, cb){
> >
> >          var self = this;
> > -        pb.cache.get(key, function(err, result){
> > +        pb.cache.get(key, function(err, result){console.log(util.inspect(result));
>
> Did you intend to commit this console.log()?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pencilblue/pencilblue/pull/620/files#r30834108>.
>